### DISCUSSION TITLE
BouncingBallEffect() -- restarts my board?

### DISCUSSION THREAD

**Discussion by 9Rune5** [PlummersSoftwareLLC/NightDriverStrip#63] [Q&A]
About the only effect I'm able to launch on my heltec_wifi_kit_32 is the PaletteEffect.

I don't have a microphone installed, so I understand that disqualifies me from quite a few (most) effects. 

But is BouncingBallEffect() supposed to work out of the box? As soon as my board starts, the TFT briefly displays the fps information (00 btw), goes black and then displays the screen again. I guess it is rebooting. It certainly isn't lighting up any LEDs.

My changes in globals.h:
#define MATRIX_WIDTH            32
#define MATRIX_HEIGHT           8
#define NUM_RINGS               0
#define RING_SIZE_0             0

platform.io:
[env:demo]                           
platform      = espressif32          
board         = esp32dev             
board         = heltec_wifi_kit_32
monitor_speed = 115200               
upload_speed  = 921600               
upload_port   = COM7                 
upload_port   = COM3                 
build_flags   = -DDEMO=1             
                -DUSE_TFT=1  
                -std=gnu++17         
                -Ofast

**Comment by laufersteppenwolf** [comment]
Any hints on the monitor of what the chip is doing?

**Reply by 9Rune5** [reply]
It reaches Draw().

If I comment out the for-loop inside Draw() it will execute (but of course without any output)
